docs/contributing/index.rst

```diff
@@ -690,3 +915,60 @@ from high-level questions to a more detailed check-list.
   instructive?
 
 - Upon merging, use the ``Rebase and Merge`` option to keep git history clean.
+
+Reporting bugs and features 
+===========================
+
+Sharing bugs and potential new features for the geomstats project is an equally significant conteibution.
```

Comment:
contribution

Reply:
How do we differentiate the other section of workflow of a contribution? I am inclined to name this something similar to bugs/features etc